### DISCUSSION TITLE
Issues Encountered During Docker Installation

### DISCUSSION THREAD

**Discussion by superstear** [xtekky/gpt4free#1321] [Q&A]
When executing docker-compose build, an error occurs during the step [2/9] RUN apt-get -qqy update && apt-get -qqy install python3 python-is-python3 pip with the message: "ERROR [2/9] RUN apt-get -qqy update && apt-get -qqy install python3 python-is-python3 pip".

**Comment by hlohaus** [comment]
Try run the image from the hub:

docker run -p 8080:80 -p 1337:1337 -p 7900:7900 --shm-size="2g" hlohaus789/g4f:latest
